workspaces/explore/.changeset/red-tigers-work.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage-community/plugin-explore-backend': patch
+'@backstage-community/plugin-explore-node': patch
+---
+
+Add ToolProviderExtensionPoint to allow setting other tool providers
```

Comment:
Please add documentation on how to use this extension point to the backend `README`.